res/values/strings.xml

```diff
@@ -187,6 +187,7 @@ there are any).</string>
     <string name="changelog_show_full">More…</string>
 
     <string name="widget_name_quickhome">Get Me Home</string>
+    <string name="shortcut_quickhome_long_label">Start search for home.</string>
```

Comment:
This string sounds a bit strange. Why didn't you just re-use the one above? Is there usually a dot at the end of these shortcuts?